config/pepperConfig.js.ctmpl

```diff
@@ -22,87 +22,116 @@ var DDP_ENV = {
     DDP_ENV['platformGAToken'] = "UA-108484823-1";
 {{end}}
 
-{{if eq $environment "dev"}}
-    DDP_ENV['auth0Domain'] = "login-dev.joincountmein.org";
-    DDP_ENV['auth0Audience'] = "ddp-dev.auth0.com";
-    DDP_ENV['basePepperUrl'] = "https://pepper-dev.datadonationplatform.org";
-{{else if eq $environment "test"}}
-    DDP_ENV['auth0Domain'] = "login-test.joincountmein.org";
-    DDP_ENV['auth0Audience'] = "ddp-test.auth0.com";
-    DDP_ENV['basePepperUrl'] = "https://pepper-test.datadonationplatform.org";
-{{else if eq $environment "staging"}}
-    DDP_ENV['auth0Domain'] = "login-staging.joincountmein.org";
-    DDP_ENV['auth0Audience'] = "ddp-staging.auth0.com";
-    DDP_ENV['basePepperUrl'] = "https://pepper-staging.datadonationplatform.org";
-{{else if eq $environment "prod"}}
-    DDP_ENV['auth0Domain'] = "login.joincountmein.org";
-    DDP_ENV['auth0Audience'] = "datadonationplatform.auth0.com";
-    DDP_ENV['basePepperUrl'] = "https://pepper.datadonationplatform.org";
+{{if ne $study_guid "testboston"}}
+    {{if eq $environment "dev"}}
+        DDP_ENV['auth0Domain'] = "login-dev.joincountmein.org";
+        DDP_ENV['auth0Audience'] = "ddp-dev.auth0.com";
+        DDP_ENV['basePepperUrl'] = "https://pepper-dev.datadonationplatform.org";
```

Comment:
@yufengwng testboston Auth0 tenant ready to roll?

Reply:
Yes, for the most part.